packages/solarwinds-apm/src/init.ts

```diff
@@ -327,6 +357,32 @@ async function metricReaders(
   return readers
 }
 
+async function logRecordProcessors(
+  config: ExtendedSwConfiguration,
+): Promise<LogRecordProcessor[]> {
+  return [
+    new BatchLogRecordProcessor(
+      new OTLPLogExporter({
+        url: config.otlp.logsEndpoint,
+        metadata: await grpcMetadata(config),
```

Comment:
Is user able to configure `http` protocol and endpoints at all, instead of the default `grpc`? Looking at this I'm thinking No, but if so was wondering if specifying `grpcMetadata` for an `http` setup would have any effect.

Reply:
No, Node.js upstream only recently started supporting configuring exporters with environment variables, so we never attempted to support it in the library. Once HTTP GetSettings is implemented I'll probably switch everything to use HTTP/Proto and stop using gRPC altogether, but I don't plan on supporting users picking one of the two unless it's ever requested explicitly.